modules/apps/object/object-test/src/testFunctional/tests/ObjectFieldsDateAndTime.testcase

```diff
@@ -127,53 +127,4 @@ definition {
 		}
 	}
 
-	@description = "LPS-143068 - Verify a custom object entry with a date and time field can be updated"
```

Comment:
We talked in person and found that we can use unit tests to verify that a switcher renders as a switcher and to verify the specifics of the label rendering.

In addition, we saw that we can create a functional test to ensure that the settings for this field are being correctly saved and applied.

Feel free to create a new task for this functional test or just add this functional test to this pull request.